distribution/lib/Standard/Base/0.0.0-dev/src/System/File/Write_Extensions.enso

```diff
@@ -50,19 +52,13 @@ polyglot java import org.enso.base.Array_Utils
 
      This allows for building the workflow without affecting the real files.
 @encoding Encoding.default_widget
-Text.write : File -> Encoding -> Existing_File_Behavior -> Problem_Behavior -> File ! Encoding_Error | Illegal_Argument | File_Error
-Text.write self path:File encoding=Encoding.utf_8 on_existing_file=Existing_File_Behavior.Backup on_problems=Problem_Behavior.Report_Warning =
+Text.write : Writable_File -> Encoding -> Existing_File_Behavior -> Problem_Behavior -> File ! Encoding_Error | Illegal_Argument | File_Error
+Text.write self (path : Writable_File) (encoding:Encoding = Encoding.utf_8) (on_existing_file:Existing_File_Behavior = Existing_File_Behavior.Backup) (on_problems:Problem_Behavior = Problem_Behavior.Report_Warning) =
```

Comment:
Probably for the linter, but we should agree on...
`(<name> : <type> = <default>)` with spaces and brackets everywhere.
This line has both with and without spaces around the `:`

Reply:
Fixed here, not sure how many other places break this rule.

We indeed are in need of a linter (Ideally on CI) to really enforce the style.